src/ceph-volume/ceph_volume/main.py

```diff
@@ -126,6 +126,18 @@ def main(self, argv):
             default='/var/log/ceph/',
             help='Change the log path (defaults to /var/log/ceph)',
         )
+        parser.add_argument(
+            '--mon-host',
+            help='List of monitors for the cluster',
+        )
+        parser.add_argument(
+            '--mon-dns-serv-name',
+            help='The name of the DNS SRV record to check to identify the cluster monitors via DNS',
+        )
+        parser.add_argument(
+            '--keyring',
```

Comment:
If you specify keyring I believe you also want `--name` too, unless you want to assume `client.admin` is always used. But then if `client.admin` is assumed, the key path should also be assumed...